SECURITY.md

```diff
@@ -0,0 +1,38 @@
+# Security policy
+
+## Supported versions
+
+Security updates will be released for all major versions that have had releases in the last year,
```

Comment:
Just to clarify "all major versions" -- given we only have 1.x.y, I guess this means just one security update?

Reply:
Yes, plus the ones that Juju needs. The "only for majors" is where we currently landed for ops, so I re-used that here.

We could have a table here listing versions like some projects do, but that seems a pain to maintain. [LXD](https://github.com/canonical/lxd/blob/main/SECURITY.md) has rules for monthly and LTS releases - we could copy that but have Juju instead of LTS?

(The "what qualifies" section in the LXD one I think would be good here too, but probably after the current modelling work is done).

Reply:
I think what you have now is fine -- I just wanted to clarify the intent. Could add a little parenthetical to clarify for others like "(1.x versions)" if you want.